apitools/base/py/exceptions.py

```diff
@@ -143,14 +182,17 @@ class RetryAfterError(HttpError):
 
     """The response contained a retry-after header."""
 
-    def __init__(self, response, content, url, retry_after):
-        super(RetryAfterError, self).__init__(response, content, url)
+    def __init__(self, response, content, url, retry_after,
+                 method_config=None, request=None):
+        super(RetryAfterError, self).__init__(
+            response, content, url, method_config, request)
         self.retry_after = int(retry_after)
 
     @classmethod
-    def FromResponse(cls, http_response):
+    def FromResponse(cls, http_response, method_config=None, request=None):
```

Comment:
Why not just `**kwargs`?

Reply:
Added **kwargs to all of the methods instead, to have single source of truth for keyword arguments in HttpError constructor. Much better, thanks.